docs/tasks/administer-cluster/weave-network-policy.md

```diff
@@ -105,6 +105,59 @@ spec:
 
 {% endcapture %}
 
```

Comment:
You need another {% capture %} section here; see the task template for the correct markup. Please check the netlify build. Right now the incorrect markup is causing the content to display out of order.